docs/using/backup.md

```diff
@@ -161,9 +161,11 @@ Make regular scheduled backups.
             - *Start time, h/m*: The hour and minute for the backup.
                 - In the first field, select one or more hours (*00* to *23*, *00* is midnight).
                 - In the second field, select one or more minutes (*00* to *59*).
+
+        - *Retention*: Number of backups to keep. Insert 0 if there is no limit.
```

Comment:
```suggestion
        - *Retention*: How many backups to keep. For unlimited, use `0` (zero).
```

Reply:
What happens when space is limited but backups are unlimited? Is there a warning?

Reply:
@Dasio  any thought about this?☝️

Reply:
No, we don't check storage. We don't know actual backup size before it is made. So it there will be not enough space, then it will fail.